databricks/koalas/namespace.py

```diff
@@ -272,14 +273,35 @@ def read_csv(
 
         reader.options(**options)
 
-        if isinstance(names, str):
-            sdf = reader.schema(names).csv(path)
-        else:
-            sdf = reader.csv(path)
-            if header is None:
-                sdf = sdf.selectExpr(
-                    *["`%s` as `%s`" % (field.name, i) for i, field in enumerate(sdf.schema)]
-                )
+        try:
+            if isinstance(names, str):
+                sdf = reader.schema(names).csv(path)
+            else:
+                sdf = reader.csv(path)
+                if header is None:
+                    sdf = sdf.selectExpr(
+                        *["`%s` as `%s`" % (field.name, i) for i, field in enumerate(sdf.schema)]
+                    )
+        except Py4JJavaError:
+            if quotechar is None:
+                quotechar = '"'
+            pdf = pd.read_csv(
+                path,
```

Comment:
Just curious; Is there a no problem with just use `pd.read_csv` directly with for large dataset?

Reply:
Good point.
Maybe we need a config to enable this or user warning.

Reply:
Totally agree.
+ And I was just thinking, How about we add the parameter like `use_pandas` for all methods for allowing users can use pandas API directly.
Because maybe sometimes(maybe for someone often) users could want to just use pandas API with small dataset without switching the library from `ks` to `pd`.
(You don't have to seriously think about this, just my opinion 😸 )

Reply:
Hmm.. just forget about above comment. maybe It's not that necessary for now.